### ISSUE TITLE
Update documentation fields names

### ISSUE THREAD

**Issue by dxcore35** [danielo515/obsidian-modal-form#243]
Please for each field that is supported write standard object name in same way as you did for:

-  input: { type: "text" }
-  input: { type: "number" }

**Comment by danielo515** [comment]
Do you mean for each field supported by modal form?

**Comment by dxcore35** [comment]
Exactly. Just to know what are the name of the variables for the supported fields...
